src/Location/Location.tsx

```diff
@@ -12,107 +12,125 @@ import wind from '../assets/wind.png'
 // feel free to break up lines of text for readability, i'm getting tired
 
 import { getAllData } from '../helpers/dataCleaner'
+import { getTestData } from '../helpers/apiCalls'
+
+// interface LocationProps {
+// 		date?: string,
+// 		day?: string,
+// 		time?: number,
+// 		city?: string;
+// 		state?: string,
+// 		country?: string,
+// 		temp?: number,
+// 		aqi?: number,
+// 		uvIndex?: number,
+// 		visibility?: number,
+// 		wind?: number,
+// 		windDirection?: string,
+// 		precipitation?: number,
+// 		allergyLevel?: number,
+// 		discussion?: string,
+//   }
 
 interface LocationProps {
-		date?: string,
-		day?: string,
-		time?: number,
-		city?: string;
-		state?: string,
-		country?: string,
-		temp?: number,
-		aqi?: number,
-		uvIndex?: number,
-		visibility?: number,
-		wind?: number,
-		windDirection?: string,
-		precipitation?: number,
-		allergyLevel?: number,
-		discussion?: string,
-  }
+	// getMatchedData: Function
+	query: string
+	matchDetails: Array<any>
+}
 
 // const Location = ({locationData, currentWeather, weeklyWeather, currentAir}) => {
-// 	const [allData, setAllData] = useState()
+const Location: React.FC<LocationProps> = props => {
+	const [allData, setAllData] = useState({})
+
+	// setAllData(getAllData('denver'))
+
+	const getMatchedData = () => {
```

Comment:
This was just so I didn't fetch data every time and can be removed as well as the call on line 53. It was my tester method!